tests/test_edgeql_data_migration.py

```diff
@@ -7714,6 +7714,88 @@ async def test_edgeql_migration_non_ddl_statements(self):
             [{'foo': 'test'}],
         )
 
+    async def test_edgeql_migration_extensions_01(self):
+        await self.con.execute('''
+            START MIGRATION TO {
+                using extension graphql;
+            };
+        ''')
+
+        await self.assert_describe_migration({
+            'confirmed': [],
+            'complete': False,
+            'proposed': {
+                'statements': [{
+                    'text':
+                        "CREATE EXTENSION graphql VERSION '1.0';"
+                }],
+                'confidence': 1.0,
+            },
+        })
+
+        await self.fast_forward_describe_migration()
+
+        await self.assert_query_result(
+            r"""
+                SELECT schema::Extension {
+                    name,
+                }
+                FILTER .name = 'graphql'
+            """,
+            [{
+                'name': 'graphql',
+            }]
+        )
+
+        await self.con.execute('''
+            START MIGRATION TO {
```

Comment:
Is this migration committing changes to the default module?

Reply:
No.  Extensions are database-wide, and so are declared at the top level.